bindings/python/iota_sdk/wallet/account.py

```diff
@@ -226,40 +226,41 @@ def pending_transactions(self):
             'pendingTransactions'
         )
 
-    def prepare_decrease_native_token_supply(self,
-                                             token_id: HexStr,
-                                             melt_amount: int,
-                                             options: Optional[TransactionOptions] = None):
+    def prepare_melt_native_token(self,
+                                  token_id: HexStr,
+                                  melt_amount: int,
+                                  options: Optional[TransactionOptions] = None):
         """Melt native tokens. This happens with the foundry output which minted them, by increasing it's
         `melted_tokens` field.
         """
         prepared = self._call_account_method(
-            'prepareDecreaseNativeTokenSupply', {
+            'prepareMeltNativeToken', {
                 'tokenId': token_id,
                 'meltAmount': hex(melt_amount),
                 'options': options
             }
         )
         return PreparedTransactionData(self, prepared)
 
-    def prepare_increase_native_token_supply(self, token_id: HexStr, mint_amount: int, options: Optional[TransactionOptions] = None):
+    def prepare_mint_native_token(self, token_id: HexStr, mint_amount: int, options: Optional[TransactionOptions] = None):
         """Mint more native token.
         """
         prepared = self._call_account_method(
-            'prepareIncreaseNativeTokenSupply', {
+            'prepareMintNativeToken', {
                 'tokenId': token_id,
                 'mintAmount': hex(mint_amount),
                 'options': options
             }
         )
         return PreparedMintTokenTransaction(account=self, prepared_transaction_data=prepared)
 
-    def prepare_mint_native_token(self, params, options: Optional[TransactionOptions] = None):
-        """Mint native token.
+    def prepare_create_native_token(self, params, options: Optional[TransactionOptions] = None):
+        """Create native token.
         """
         prepared = self._call_account_method(
-            'prepareMintNativeToken', {
-                'params': params,
+            'prepareCreateNativeToken', {
+                'tokenId': token_id,
+                'mintAmount': hex(mint_amount),
```

Comment:
That seems incorrect

Reply:
yeah :shrug:

Reply:
Fixed (probably wasn't careful enough during a merge)